flux/stdlib/influxdata/influxdb/rules.go

```diff
@@ -551,3 +566,55 @@ func isPushableFieldOperator(kind ast.OperatorKind) bool {
 
 	return false
 }
+
```

Comment:
This code must be taken from 2.0 OSS?

Reply:
Yes I sync this directory from 2.0 OSS when I do an update.